chat/QuickJump.vue

```diff
@@ -0,0 +1,459 @@
+          break;
+
+        case Keys.Enter:
+          e.preventDefault();
+          if (
+            this.selectedIndex === -1 &&
+            this.filteredResults.length === 0 &&
+            this.searchQuery.length > 0
+          ) {
+            // Open new conversation
+            this.openNewConversation();
+          } else if (this.filteredResults.length > 0) {
+            this.selectResult(this.filteredResults[this.selectedIndex]);
+          }
+          break;
+      }
+    }
+
+    selectResult(result: SearchResult): void {
+      if (result.conversation) {
+        result.conversation.show();
+      }
+      this.hide();
+    }
+
+    openNewConversation(): void {
+      if (this.searchQuery.trim().length === 0) return;
+
+      // Create a character object for the username
+      const character = core.characters.get(this.searchQuery.trim());
+
+      // Create and show the conversation
+      const conversation = core.conversations.getPrivate(character);
+      conversation.show();
+
+      this.hide();
+    }
+
+    @Watch('visible')
+    onVisibilityChange(newValue: boolean): void {
+      if (newValue) {
+        // Update results when showing
+        this.updateResults();
+      }
+    }
+  }
+</script>
+
+<style lang="scss" scoped>
+  .quick-jump-overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    z-index: 9999;
+    display: flex;
+    align-items: flex-start;
+    justify-content: center;
+    padding-top: 10vh;
+
+    .quick-jump-container {
+      background-color: var(--backgroundColor);
+      border: 1px solid var(--borderColor);
+      border-radius: 12px;
+      box-shadow:
+        0 20px 60px rgba(0, 0, 0, 0.3),
+        0 8px 25px rgba(0, 0, 0, 0.15);
+      width: 90%;
+      max-width: 600px;
+      overflow: hidden;
+      backdrop-filter: blur(8px);
+      -webkit-backdrop-filter: blur(8px);
+
+      .quick-jump-header {
+        background-color: var(--primaryColor);
+        color: var(--primaryTextColor);
+        padding: 12px 16px;
+        font-weight: bold;
+        display: flex;
+        align-items: center;
+        gap: 8px;
+      }
+
+      .quick-jump-input {
+        width: 100%;
+        border: none;
+        padding: 16px;
+        font-size: 16px;
+        background-color: var(--backgroundColor);
+        color: var(--textColor);
+        outline: none;
+
+        &::placeholder {
+          color: var(--gray);
+        }
+      }
+
+      .quick-jump-results {
+        max-height: 400px;
+        overflow-y: auto;
+        border-top: 1px solid var(--borderColor);
+
+        .quick-jump-result {
+          padding: 12px 16px;
+          display: flex;
+          align-items: center;
+          gap: 12px;
+          cursor: pointer;
+          border-bottom: 1px solid rgba(255, 255, 255, 0.1);
+          transition: all 0.2s ease;
+          position: relative;
+
+          &:hover {
+            background-color: rgba(59, 130, 246, 0.1);
```

Comment:
Preferably this is a color based on the theme too, could always just use a SASS function to get a more subtle version of the primary color.